dist/token-autocomplete.css

```diff
@@ -139,3 +139,31 @@
     font-size: 0.7em;
     color: #808080;
 }
+
+.token-autocomplete-container.optional-singleselect-with-value {
+    position: relative;
+    padding-right: 20px;
+}
+
+.token-autocomplete-container.token-autocomplete-singleselect.optional-singleselect-with-value .token-autocomplete-input:after {
+    right: -20px;
+}
+
+.token-autocomplete-container.token-autocomplete-singleselect.optional-singleselect-with-value .token-singleselect-token-delete {
+    content: 'x';
+    color: #000;
+    position: absolute;
+    top: 10px;
+    right: 20px;
+    background-color: #cccccc;
+    border-radius: 20px;
+    padding-left: 4px;
+    padding-right: 4px;
+    padding-bottom: 2px;
+    cursor: pointer;
+    display: block;
+}
```

Comment:
Is this a generic minimalistic default style or is this already the look and feel of the old BS3 select?

Reply:
<img width="150" alt="Screenshot 2021-06-30 at 10 54 12" src="https://user-images.githubusercontent.com/10437176/123932270-91879800-d991-11eb-9f3e-3e289be3ae8b.png">

Reply:
perfect, thx